guides/common/assembly_configuring-smartproxyservers-with-custom-ssl-certificates-for-load-balancing-with-puppet.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/con_configuring-smartproxyservers-with-custom-ssl-certificates-for-load-balancing-with-puppet.adoc[]
+
+:parent-context: {context}
+:context: LB-custom-ssl-with-puppet 
+include::modules/ref_prerequisites-configuring-smartproxyservers-for-load-balancing.adoc[]
+
```

Comment:
```suggestion
```
The others as well.

Reply:
Btw, shouldn't this module have offset +1?

Reply:
@Lennonka It looks like the missing leveloffset was the culprit all along. The book builds without the blank lines now that I've fixed that.